src/react-app/components/landing/RegistrationStep.js

```diff
@@ -9,20 +12,24 @@ const RegistrationStep = (props) => {
     color,
     description,
     Icon,
+    iconsrc,
   } = props;
 
   const isSmall = isSmallDevice();
   return (
     <Col 
-      className={`background-${color} text-${color} border-round padding-half--sides padding-half--ends margin--top`} 
+      className={`border-round padding-half--sides padding-half--ends margin-double--top`} 
       offset={(!isSmall && number!==1) ? 1 : 0} 
       xs={24} 
       sm={24} 
       md={6}
     >
-      <div className={`margin--bottom ${isSmall ? 'f24' : 'f48'}`}>{number}</div>
-      <Icon className={`center text-${color} margin-double--bottom ${isSmall ? 'f72' : 'f108'}`} style={{display:'table', margin:'0 auto'}}/>
-      <div className='f16 margin--ends margin--sides'>{description}</div>
+      <div className={`margin--bottom ${isSmall ? 'f24' : 'f48'} stepnum-style`}>{number}</div>
+      {/* <Icon className={`center margin-double--bottom ${isSmall ? 'f72' : 'f108'}`} style={{display:'table', margin:'0 auto'}}/> */}
+      <span className={`center margin-double--bottom ${isSmall ? 'f72' : 'f108'}`} style={{display:'table', margin:'0 auto'}}>
+        <img src={`${(number == 1) ? info : (number == 2) ? notif : data}`} className="width-100"/>
```

Comment:
why not just use iconsrc? that should give the source of this you can put the assets import as the icon src in the utils